.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Are you able to remove the `.idea` directory?

Reply:
Certainly.